styles/mandar.css

```diff
@@ -631,6 +632,43 @@ img {
   border-image: linear-gradient(to right, rgba(0, 0, 0, 0), rgba(73, 12, 11, 0.85), rgba(0, 0, 0, 0)) 5%;
 }
 
+/* confirmed good */
+.starwarsffg.sheet .sheet-body.gear {
+  height: calc(100% - 24rem);
+}
+
+.starwarsffg.sheet .sheet-body.weapon, .starwarsffg.sheet .sheet-body.armour {
+  height: calc(100% - 32rem);
+}
+
+.starwarsffg.sheet .sheet-body.shipweapon {
+  height: calc(100% - 40rem);
+}
+
+.starwarsffg.sheet .sheet-body.itemattachment {
+  height: calc(100% - 18rem);
+}
+
+.starwarsffg.sheet .sheet-body.ability {
+  height: calc(100% - 1.5rem);
+}
+
+.starwarsffg.sheet .sheet-body.shipattachment {
+  height: calc(100% - 22.75rem);
+}
+
+.starwarsffg.sheet .sheet-body.homesteadupgrade {
+  height: calc(100% - 20rem);
+}
+
+.starwarsffg.sheet .sheet-body.talent {
+  height: calc(100% - 26rem);
+}
+
+.starwarsffg.sheet .sheet-body.species {
+  height: calc(100% - 24rem);
+}
+

```

Comment:
Gonna approve this PR cause I don't wanna slow you down, but bear in mind this could easily become nightmareish to maintain 😂

Reply:
Agreed. Perhaps we can chat on a non-stupid way of doing it. This is what I found works, but it doesn't appear like the _correct_ solution, even to me.